packages/js/src/utils/stop-local-registry.ts

```diff
@@ -0,0 +1,14 @@
+import { execSync } from 'child_process';
+
+export function stopLocalRegistry() {
```

Comment:
If the above is possible, this doesn't need to be a separate function.

Reply:
removed